### PR TITLE
Use fetch() function to display Servlet message

### PR DESCRIPTION
**Pull request by jonaner**
Screenshot 2020-06-03 at 11 54 40 AM - Display 2
Screenshot 2020-06-03 at 11 55 16 AM - Display 2

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -4,18 +4,21 @@
...
     <title>Jonathan Aner's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
       <h1>Jonathan's Portfolio</h1>
       <h2>Welcome to my Portfolio!</h2>
+      <button onclick="addGreeting()">Fetch Greeting</button>
+      <h1><div id="quote-container"></div></h1>
```
Comment (jbir789) [inline portfolio/src/main/webapp/index.html:14]: Why did you decide to use a `<div>` over `<span>`?

File: portfolio/src/main/webapp/personalblog.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+        <!DOCTYPE html>
```
Comment (jbir789) [inline portfolio/src/main/webapp/personalblog.html:1]: nit: remove the leading space

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,8 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addGreeting(){
+    fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('quote-container').innerText = quote;
+    });
+}
```
Comment (jbir789) [inline portfolio/src/main/webapp/script.js:33]: nit: please add an empty line at the end of the file. Here's an interesting read on why this is a convention in the software development world: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

File: portfolio/src/main/webapp/personalblog.html (hunk 2 above)
Comment (jonaner) [reply portfolio/src/main/webapp/personalblog.html:1]: I was following the same formatting that I saw in the week 3 project folder, but span does make more sense.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jonaner) [reply portfolio/src/main/webapp/index.html]: I was following the same format I saw in the week 3 folder, but span does make sense.

**Comment by jbir789** [comment]
I see, thanks!

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addGreeting(){
+    fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('quote-container').innerText = quote;
```
Comment (jbir789) [inline portfolio/src/main/webapp/script.js:32]: nit: indent this line since it's between curly brace

**Comment by jbir789** [comment]
Please make sure to require 2 approvals before merging, using the instructions at: https://help.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests

File: walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingAsyncAwait()">Fetch a quote!</button>
```
Comment (ginellegaisano) [inline walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: I don't see #getRandomQuoteUsingAsyncAwait in this PR - please include the function as well.

Comment (jonaner) [reply walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html:13]: This index.html is located in the week-3-server random-quote directory. I had to change it around to see the differences in how suggested ways to write the fetch() method. The getRandomQuoteUsingAsyncAwait() method is located in that directory's script.js. No changes were made to that specific script.js.

**Pull request merged**